bitbots_dynup/msg/DynupEngineDebug.msg

```diff
@@ -0,0 +1,21 @@
+#
+# This message is only used for debugging the dynup engine.
+#
+
+std_msgs/Header header
+
+float64 time
+bool stabilization_active
+int32 state_number
```

Comment:
The state numbers might be an enum in the message, that would increase readability in `DynupEngine::publish_debug`.